### ISSUE TITLE
Exclude h2 from war

### ISSUE THREAD

**Issue by todoubaba** [grails/grails-core#9919]
### Steps to Reproduce

`> grails create-app test`
delete line `runtime "com.h2database:h2"` from build.gradle
`> grails package`

### Expected Behaviour

h2-1.4.191.jar is included in test.war.

### Actual Behaviour

exclude unnecessary jar file.

### Environment Information

- **Operating System**: win 10
- **Grails Version:** 3.1.6
- **JDK Version:** 1.8

**Comment by graemerocher** [comment]
h2 is included in the default set of dependencies because the default database configuring uses. If you don't want it then you must exclude it from `grails-dependencies` too:

```groovy
    compile "org.grails:grails-dependencies", {
        exclude group:'com.h2database', module:'h2'
    }
```

**Comment by todoubaba** [comment]
Yes, H2 is the default database, but it is also included in the profile's base hibernate feature. Is it duplicate to include it in both `grails-dependencies` and the profile?

**Comment by todoubaba** [comment]
Fixed in Grails 3.3.0
